ecosystem/sep_identicons.md

```diff
@@ -0,0 +1,97 @@
+## Preamble
+
+```
+SEP: To Be Assigned
+Title: Identicons for Stellar Accounts
+Author: Lobstr.co
+Track: Informational
+Status: Draft
+Created: 2019-11-20
+Discussion: https://groups.google.com/forum/#!topic/stellar-dev/kAqhpCMe96c
+```
+
+## Simple Summary
+This SEP defines a way to generate identicons for Stellar accounts - unique icons, based on the public key of the account.  
+It's a visual representation of the Stellar address, that serves to identify the account as a form of an avatar.
+
+As an example, here's the identicon for this Stellar account `GALAXYVOIDAOPZTDLHILAJQKCVVFMD4IKLXLSZV5YHO7VY74IWZILUTO`:
+
+![image](https://id.lobstr.co/GALAXYVOIDAOPZTDLHILAJQKCVVFMD4IKLXLSZV5YHO7VY74IWZILUTO.png)
+
+
+## Motivation
+The goal is to define a standard way to generate identicons for Stellar addresses.
+
+Having a standard algorithm and readily available tools to generate identicons would simplify the integration for the service providers. At the same time, end users would benefit from seeing the same icon for their Stellar account in different projects across the ecosystem.
+
+
+## Abstract
+Identicon for a Stellar account is an image generated based on a public key ("G...").
+
+Identicons can be used in the interface of products built on Stellar to improve the user experience, because they:
+- allow users to visually distinguish different addresses;
+- may help to verify that the sending of funds is done to correct account;
+- may help to visualize history of transactions or multisig configuration;
+- may help to identify which account user is currently logged in with;
+
+This SEP describes the algorithm used to generate the images and provides links to several reference implementations.
+
+Identicons should be used only as an additional tool in the application UI, to provide more context to the user and help prevent errors. A full public address should still be visible to the user and available for verification.
+
+## Specification
+
+The generator takes a public key ("G...") as an input.
+
+First, the public key is converted to a sequence of bytes, in a manner identical to the `keypair.rawPublicKey()` function of JS Stellar SDK. A portion of bytes from the raw public key in the position from 2 to 16 are used for the identicon generation (`keypair.rawPublicKey().slice(2,16)`), while the remaining bytes are ignored.
```

Comment:
[Edit: I've removed the above comment because I think debating how the icon is generated is irrelevant here as this identicon format is already in use in the ecosystem, if there's ways to improve it they should probably be in a new SEP or a new version/proposal for this SEP.]